app/controllers/game_controller.rb

```diff
@@ -1,40 +1,46 @@
 class GameController < ApplicationController
   def new
-    render 'new'
+    render "new"
   end
 
   def init
-    session['board'] = @board = Board.new.matrix
+    session["board"] = @board = Board.new.matrix
     @players = PlayerList.new(params[:x], params[:o])
-    session['players'] = @players.players
-    session['current_player'] = @current_player = @players.current_player
+    session["players"] = @players.players
+    session["current_player"] = @current_player = @players.current_player
   end
 
   def turn
-    @board = Board.new session['board']
-    @players = PlayerList.new session['players'][0]['name'], session['players'][1]['name'], session['current_player']
+    @board = Board.new session["board"]
+    @players = PlayerList.new session["players"][0]["name"],
+                              session["players"][1]["name"],
+                              session["current_player"]
     @coordinates = get_coordinates(params[:id]) #if id != nil or ""
-    @symbol = @players.players[session['current_player'].to_i].symbol
+    @symbol = @players.players[session["current_player"].to_i].symbol
 
     @finished = play @board, @coordinates, @symbol
     @players.next!
-    session['players'] = @players.players
-    session['current_player'] = @players.current_player
+    session["players"] = @players.players
+    session["current_player"] = @players.current_player
 
     respond_to do |format|
-      format.json { render json: { board: @board, players: @players, current_player: @players.current, finished: @finished } }
+      format.json do
+        render json: {
+                   board: @board, players: @players, current_player: @players.current, finished: @finished
```

Comment:
Line is too long. [106/80]<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.